packages/base/src/mainview.tsx

```diff
@@ -125,9 +132,11 @@ export class MainView extends React.Component<IProps, IStates> {
       this
     );
 
-    if (this._raycaster.params.Line) {
-      this._raycaster.params.Line.threshold = 0.1;
-    }
+    // @ts-ignore Missing ThreeJS typing
```

Comment:
Upgrading `threejs` might help? It looks like the `Line2` property is added to the new interface
 https://github.com/three-types/three-ts-types/blob/056281e10e2b0aa0f6f2a126f9020b60e7d45759/types/three/src/core/Raycaster.d.ts#L42

Reply:
Indeed, I'm a bit worried about updating ThreeJS though, would it be fine for you doing this in a separate PR?

Reply:
Yes